src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowDiagnosticTests.cs

```diff
@@ -1056,6 +1056,31 @@ partial struct S2
                 );
         }
 
+        [Fact]
+        [WorkItem(23668, "https://github.com/dotnet/roslyn/issues/23668")]
+        public void PartialWithPropertyButSingleField()
```

Comment:
>PartialWithPropertyButSingleField [](start = 20, length = 33)

It doesn't feel like this test belongs in this file. The CS0282 warning is not reported by flow analysis.

Reply:
Moved it to SymbolErrorTests.cs where the other test for that warning is.